src/www/js/events/events.js

```diff
@@ -13,3 +13,23 @@
 
 
 // Your code here.
+
+
+const list = document.getElementsByTagName('li')
+
+list[0].firstElementChild.addEventListener('click', () => { increment(list[0]) } )
+list[1].firstElementChild.addEventListener('click', () => { increment(list[1]) } )
+
+const plist = document.getElementsByTagName('p')
+
+plist[0].firstElementChild.addEventListener('click', () => { increment(plist[0]) } )
+plist[1].firstElementChild.addEventListener('click', () => { increment(plist[1]) } )
+plist[2].firstElementChild.addEventListener('click', (e) => { console.log(e)
+    e.preventDefault
+    increment(plist[2]) } )
+
+function increment(obj){
+    val = parseInt(obj.lastElementChild).innerHTML
+    console.log(val);
+    obj.lastElementChild.innerHTML = (val+1).toString();
+}
```

Comment:
Nice start. :-) I'd suggest you try working with `innerText` because then you don't need to worry about HTML sneaking in there.

Also you can try using event delegation rather than putting the event handlers on each child, as we talked about in the class. Remember to use `event.target`. Would be a good change to do.

The main issue with your current approach is that it's not abstracted, and the logic would break if we added additional `p` tags or whatever. See if you can make even make a single event handler that does everything, that would be fun challenge. :-)